docs/README.md

```diff
@@ -0,0 +1,21 @@
+# Repository
+
+This folder contains the documentation for Chipmunk compiler.
+
+# Compiler source code organization
+
+```
+
+chipmunk-tofino
+├── chipc
+│   ├── lib                   -- library files including antlr version 4.7.2
+│   └── templates             -- collection of jinja2 template files
+├── chipc.egg-info            -- some egg info
```

Comment:
Remove this

https://github.com/chipmunk-project/chipmunk-tofino/blob/2dac70e2b36356102706c23c6c684088f60dae06/.gitignore#L16

*.egg-info directory is ignored by Git, no need to document. 
btw, egg-info contains project metdadata: https://setuptools.readthedocs.io/en/latest/formats.html#eggs-and-their-formats

Reply:
Sure, I will remove it. As for lib folder, is it used to include all dependency library files needed for Chipmunk?

Reply:
Yes